### DISCUSSION TITLE
is this course finished?

### DISCUSSION THREAD

**Discussion by UrDailyCs** [mrdbourke/pytorch-deep-learning#122] [General]
Hey, i see you have finished recording all videos for this course. wonder when Section 8 and 9 will come out? can we expect it this month? also i understand that you going to release this course on Udemy after completing all sections, can we expect it this month too or will it be next month?

**Comment by mrdbourke** [comment]
Hi @UrDailyCs,

Yes all of the course videos have been finished recording (as of September 2022).

They are being edited as we speak and will be uploaded to ZTM as soon as possible.

Once the full release is published on ZTM, you can expect it to come to Udemy.

**Reply by gdineshk6174** [reply]
hi, @mrdbourke ,any idea when the complete course will be in Udemy?

**Reply by mrdbourke** [reply]
very soon :)

**Comment by mrdbourke** [comment]
Hi @UrDailyCs and @gdineshk6174,

Good news! The course is now available in full on Udemy, you can sign up here: https://www.udemy.com/course/pytorch-for-deep-learning/?couponCode=ZTMGOODIES7 (discount code "ZTMGOODIES7" applied for the next ~4 days to get the best price).
